go/vm/world_state.go

```diff
@@ -62,13 +64,41 @@ type StorageStatus int
 
 // See t.ly/b5HPf for the definition of these values.
 const (
-	StorageAssigned StorageStatus = iota
-	StorageAdded
-	StorageDeleted
-	StorageModified
-	StorageDeletedAdded
-	StorageModifiedDeleted
-	StorageDeletedRestored
-	StorageAddedDeleted
-	StorageModifiedRestored
+	// The comment indicates the storage values for the corresponding
+	// configuration. X, Y, Z are non-zero numbers, distinct from each other,
+	// while 0 is zero.
+	//
+	StorageAssigned         StorageStatus = iota
```

Comment:
This line should be moved as well:
```suggestion
	//
	// <original> -> <current> -> <new>
	StorageAssigned         StorageStatus = iota
```